drivers/sensor/sensor_shell.c

```diff
@@ -106,6 +113,38 @@ static const char *sensor_attribute_name[SENSOR_ATTR_COMMON_COUNT] = {
 	[SENSOR_ATTR_FF_DUR] = "ff_dur",
 };
 
+/* Forward declaration */
+static void data_ready_trigger_handler(const struct device *sensor,
+				       const struct sensor_trigger *trigger);
+
+#define TRIGGER_DATA_ENTRY(enum, str_name, handler_func)                                           \
```

Comment:
using a keyword for a macro param is a bit confusing here, can we change this to be something like trig_type or so